templates/logrotate.rails

```diff
@@ -0,0 +1,10 @@
+/srv/**/log/*.log {
+  su root root
+  daily
+  size 50M
```

Comment:
i'm guessing we are leaving it up to users to decide if they need variables? i.e if I have a template that expects
```jinja
 size {{ logrotate_max_size }} 
```
I'd have to define `logrotate_max_size` in my playbook?